addon/helpers/flatten.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+
+const {
+  Helper,
+  computed,
+  get,
+  isArray,
+  observer,
+  set
+} = Ember;
+
+export function flatten(array) {
+  array = array || [array];
+
+  return array.reduce((flat, toFlatten) => {
+    return flat.concat(Array.isArray(toFlatten) ? flatten(toFlatten) : toFlatten);
```

Comment:
Can we use `Ember.typeOf` here? Just so that other kinds of Ember Arrays will also be true here.